x-pack/plugins/ml/public/application/components/links/ml_href.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { ReactChild } from 'react';
+import { useMlKibana, useNavigateToPath } from '../../contexts/kibana';
+
+function isModifiedEvent(event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) {
+  return !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey);
+}
+
+export const MlHref = ({
+  href,
+  target,
+  onClick,
+  children,
+  ...rest
```

Comment:
Not sure if `...rest` should be here because you already destructured all the props.

Reply:
This `...rest` is here if the user wants to pass on extra props for the <a> tag like 'className' or `style`.